.github/workflows/pr-check.yml

```
@@ -19,7 +19,7 @@ jobs:
       - name: Check Type
         uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '^(build|ci|docs|feat|fix|refactor|test)(|!): [A-Za-z].+?((, fixes #\d+)*)($|\n|\r|\n\r)'
+          pattern: '^(build|chore|ci|docs|feat|fix|perf|refactor|style|test)(\([a-z0-9-]+\))?(!)?:\s[a-z].+((,\s+fixes\s+#\d+)*)$'
```

revert for uppercase in the message:

```suggestion
          pattern: '^(build|chore|ci|docs|feat|fix|perf|refactor|style|test)(\([a-z0-9-]+\))?(!)?:\s[A-Za-z].+((,\s+fixes\s+#\d+)*)$'
```

---

I'm unable to find a proper regex for `, fixes`.

---

`, fixes #[0-9]+$` would do it, if that's the goal